src/Domain/Entities/Product/ValueObjects/ProjectTitleValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CleanCompanyName.DDDMicroservice.Domain.Common.Validators;
+using CleanCompanyName.DDDMicroservice.Domain.Entities.Product.ValueObjects;
+using FluentValidation;
+
+namespace CleanCompanyName.DDDMicroservice.Domain.Entities.Product.ValueObject;
+
+public class ProjectTitleValidator : AbstractValidator<ProjectTitle>
+{
+    public const int MinLength = 5;
+    public const int MaxLength = 350;
+
+    public ProjectTitleValidator()
+    {
+        RuleFor(projectTitle => projectTitle.Value)
+            .NotEmpty()
+            .WithErrorCode("Title_NotEmpty")
+            .MinimumLength(MinLength)
+            .WithCustomErrorCode(this)
```

Comment:
Here we have an example of how to let our custom validator extension write our code error.